package.json

```diff
@@ -28,6 +28,7 @@
     "ejs": "^3.1.6",
     "express": "4.17.1",
     "form-data": "3.0.0",
+    "link-preview-js": "^2.1.8",
```

Comment:
remove ^ so that it doesn't use the latest version which might go without the testing